frontend/src/metabase/palette/components/PaletteResultItem.tsx

```diff
@@ -117,14 +117,7 @@ export const PaletteResultItem = ({
       </Flex>
       {/** Active container */}
       {active && (
-        <Flex
-          aria-hidden
-          gap="0.5rem"
-          fw={400}
-          style={{
-            flexBasis: 60,
-          }}
-        >
+        <Flex aria-hidden gap="0.5rem" fw={400}>
```

Comment:
This change isn't related to the search result, but it lets the "open" text container grow / shrink while still laying out correctly. We noticed that in other languages, the text could wrap making things look bad.